### ISSUE TITLE
Divide by zero error

### ISSUE THREAD

**Issue by darreldonnelly** [fivetran/dbt_pendo#15]
Hi there, 

We are using the pendo package for metrics aggregation but running into an issue with a divide by zero in the intermediate model below:

https://github.com/fivetran/dbt_pendo/blob/04f1385b96417ffc62570674d88cd7285e6ea8e8/models/intermediate/daily_metrics/int_pendo__page_daily_metrics.sql#L64

There are a few divisions done with 'numerator / denominator' which is causing us an issue in our dbt models running on snowflake (we're currently in ramp up on pendo so low volume, this might not be noticed in busy environments) 

```
[2022-08-24, 19:53:14 UTC] {pod_manager.py:226} INFO - 19:53:14  Database Error in model int_pendo__page_daily_metrics (models/intermediate/daily_metrics/int_pendo__page_daily_metrics.sql)
[2022-08-24, 19:53:14 UTC] {pod_manager.py:226} INFO - 19:53:14    100051 (22012): Division by zero
[2022-08-24, 19:53:14 UTC] {pod_manager.py:226} INFO - 19:53:14    compiled SQL at target/run/pendo/models/intermediate/daily_metrics/int_pendo__page_daily_metrics.sql
```

I'd be happy to raise a PR but am not 100% sure of style or cross database compatibility requirements. On snowflake this could be a DIV0 but i'm guessing we'd need to do something like this for maximum compatibility:

```sql
 round(100.0 * sum_pageviews / NULLIF(total_pageviews,0), 3) as percent_of_daily_pageviews, 
```

**Comment by fivetran-joemarkiewicz** [comment]
Hi @darreldonnelly thanks so much for raising this issue. I can see from your description how this could result in failures across the package users. I think you suggestion makes sense and is one we can incorporate within the package!

In terms of cross database compatibility, we have used `nullif` across our other packages and have found it works across the supported databases. If you would like to open a PR, myself or my team can review and work to merge your contribution into the next release of the Pendo package!

**Comment by fivetran-joemarkiewicz** [comment]
Hi @darreldonnelly just wanted to post back that my team is happy to take this issue on in our upcoming sprint (starting Wednesday of this week). However, if you wanted to open a PR we would be happy to work with you on it. Let us know, otherwise we will work to implement the update in the coming weeks! 😄

**Comment by floydmcboyd** [comment]
We're seeing this as well now in our pipelines. Any ETA on a fix? (not sure your sprint cadence)

**Comment by fivetran-reneeli** [comment]
Hi @floydmcboyd ! This is in our current sprint, which ends 9/21, so expect an update by tomorrow!

**Comment by fivetran-reneeli** [comment]
Hi @darreldonnelly and @floydmcboyd, thanks for raising this issue! We've added the fix in our latest release [v0.2.1](https://github.com/fivetran/dbt_pendo/releases/tag/v0.2.1). We'll be closing this but let us know if there's anything else!
